addons/google_gmail/models/google_gmail_mixin.py

```diff
@@ -72,7 +72,7 @@ def open_google_gmail_uri(self):
         if not self.google_gmail_uri:
             raise UserError(_('Please configure your Gmail credentials.'))
 
-        if not self.smtp_user:
+        if not self.user:
```

Comment:
Definitely should be smtp_user, it's just that we need to get it from the mail server(s)

Reply:
Was about to submit my own PR^